mycroft/skills/__main__.py

```diff
@@ -275,6 +275,7 @@ def _initialize_skill_manager(bus, watchdog):
     """
     skill_manager = SkillManager(bus, watchdog)
     skill_manager.load_priority()
+    skill_manager.load_plugin_skills()
```

Comment:
This means plugin skills bypass the `wait_for_internet` optional check. Would it make sense to put this call inside `skill_manager.run()` to match current behavior of non-plugin skills?

Reply:
the wait for internet check was really meant for msm to download and update skills... since #25 this is not even a concern anymore. 

Doing this before internet improves load times considerably

Reply:
I guess there's no backwards-compat issue since plugin skills are new, but existing skills might not account for no internet connection on startup

Reply:
skills should never assume internet is present, since ovos is meant to run offline

internet dependency is pretty much a remnant of selene being mandatory, but we dont even need pairing in ovos-core

individual skills should account for this, internet could go out after boot regardless!

I'm still moving the method for consistency, but this is not a solution for the problem you are worried about

Reply:
Fair enough, never said it was a good assumption :stuck_out_tongue_winking_eye:. At some point, maybe we can define online access as a dependency or something per-skill